tests/foreman/ui/test_repositories.py

```diff
@@ -0,0 +1,61 @@
+"""Unit tests for the new ``repositories`` paths.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:team: Phoenix-content
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9])
+def test_positive_override_custom_products_on_existing_host(
+    session,
+    default_location,
+    setup_content,
+    rhel_contenthost,
+    target_sat,
+):
+    """Verify that custom products can be easily enabled/disabled on existing host
+    using "select all" method
+
+    :id: a3c9a8c8-a40f-448b-961a-7eb888883ba9
+
+    :steps:
+        1. Create custom product and upload repository
+        2. Attach to activation key
+        3. Register Host
+        4. Assert that custom proudcts are disabled by default
+        5. Override custom products to enabled using new functionality
+        6. Assert custom products are now enabled
+
+    :expectedresults: Custom products should be easily enabled.
+    """
+    ak, org, custom_repo = setup_content
+    client = rhel_contenthost
+    client.install_katello_ca(target_sat)
```

Comment:
Please use the global registration template for the registration part as this one is deprecated.
https://redhat-internal.slack.com/archives/C04T3JZ5W8K/p1685545990914279